src/Search/Console.php

```diff
@@ -120,6 +120,10 @@ public function __construct(Application $console, Kernel $app)
         $this->app = $app;
         $this->root = __DIR__.'/../..';
 
+        if (!defined('GEARMAN_INSTALLED')) {
```

Comment:
We could change how it works, but currently when the Console class is constructed it creates all of the other commands, which in turn creates a Gearman client and looks for a Gearman daemon, the only drawback is a PHP notice saying "no gearman instances found" but I thought I should detect its existence and not construct gearman if its not installed. Maybe it would be better to have gearman be "lazy"